metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Dyanmically generates /etc/ssh/ssh_known_hosts based on search indexes'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '4.1.0'
+version           '4.2.0'
```

Comment:
Sorry, this was not supposed to be part of the PR. We had released a new version internally since the PR was created.